pkg/reconciler/revision/resources/deploy.go

```diff
@@ -50,6 +50,25 @@ var (
 		SubPathExpr: "$(K_INTERNAL_POD_NAMESPACE)_$(K_INTERNAL_POD_NAME)_",
 	}
 
+	varProjectedVolume = corev1.Volume{
+		Name: "knative-proj-volume",
+		VolumeSource: corev1.VolumeSource{
+			Projected: &corev1.ProjectedVolumeSource{
+				Sources: []corev1.VolumeProjection{{
+					ServiceAccountToken: &corev1.ServiceAccountTokenProjection{
+						ExpirationSeconds: ptr.Int64(600),
+						Path:              "token",
```

Comment:
we may want to call this `state-token`, just in case we ever want to mount a token with a different audience under `/var/run/secrets/token`. This is consistent with the example in https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection too.